src/bindings/python/flux/job/list.py

```diff
@@ -96,12 +103,22 @@ def get_job(self):
         return self.get()["job"]
 
     def get_jobinfo(self):
+        """
+        Returns a JobInfo object for the job.
+        """
         return JobInfo(self.get_job())
 
 
 # list-id is not like list or list-inactive, it doesn't return an
 # array, so don't use JobListRPC
 def job_list_id(flux_handle, jobid, attrs=["all"]):
+    """Watch a job by providing an ID. 
```

Comment:
I don't think `Watch` is the right verb here. This is just a job listing that function lists a single job by id. Perhaps just "Query job information for a single jobid".